README.md

```diff
@@ -4,41 +4,56 @@
   <img alt="caustics logo" src="media/caustics_logo.png" width="70%">
 </picture>
 
-
+[![ssec](https://img.shields.io/badge/SSEC-Project-purple?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA0AAAAOCAQAAABedl5ZAAAACXBIWXMAAAHKAAABygHMtnUxAAAAGXRFWHRTb2Z0d2FyZQB3d3cuaW5rc2NhcGUub3Jnm+48GgAAAMNJREFUGBltwcEqwwEcAOAfc1F2sNsOTqSlNUopSv5jW1YzHHYY/6YtLa1Jy4mbl3Bz8QIeyKM4fMaUxr4vZnEpjWnmLMSYCysxTcddhF25+EvJia5hhCudULAePyRalvUteXIfBgYxJufRuaKuprKsbDjVUrUj40FNQ11PTzEmrCmrevPhRcVQai8m1PRVvOPZgX2JttWYsGhD3atbHWcyUqX4oqDtJkJiJHUYv+R1JbaNHJmP/+Q1HLu2GbNoSm3Ft0+Y1YMdPSTSwQAAAABJRU5ErkJggg==&style=plastic)](https://escience.washington.edu/wetai/)
 [![tests](https://github.com/Ciela-Institute/caustics/actions/workflows/python-app.yml/badge.svg?branch=main)](https://github.com/Ciela-Institute/caustics/actions)
 [![Docs](https://github.com/Ciela-Institute/caustics/actions/workflows/documentation.yaml/badge.svg)](https://github.com/Ciela-Institute/caustics/actions/workflows/documentation.yaml)
 [![PyPI version](https://badge.fury.io/py/caustics.svg)](https://pypi.org/project/caustics/)
-[![coverage](https://img.shields.io/codecov/c/github/Ciela-Institute/caustics)](https://app.codecov.io/gh/Ciela-Institute/caustics)
+[![coverage](https://img.shields.io/codecov/c/github/Ciela-Institute/caustic)](https://app.codecov.io/gh/Ciela-Institute/caustic)
+
 # caustics
 
-The lensing pipeline of the future: GPU-accelerated, automatically-differentiable,
-highly modular. Currently under heavy development: expect interface changes and
-some imprecise/untested calculations.
+The lensing pipeline of the future: GPU-accelerated,
+automatically-differentiable, highly modular. Currently under heavy development:
+expect interface changes and some imprecise/untested calculations.
 
-## Installation 
+## Installation
 
 Simply install caustics from PyPI:
+
 ```bash
 pip install caustics
 ```
 
 ## Documentation
 
-Please see our [documentation page](Ciela-Institute.github.io/caustics/) for more detailed information.
+Please see our [documentation page](Ciela-Institute.github.io/caustics/) for
```

Comment:
We might want to change this to the readthedocs, if that is going to be the primary docs source going forward